Combat.md

```diff
@@ -1,5 +1,63 @@
+
+|Action Type|AP Cost|
+|---|---|
+|[Instant Action]|0 AP|
+|[Fast Action]|1 AP|
+|[Standard Action]|2 AP|
+|[Slow Action]|3 AP|
+|[Immediate Action]|0 AP*|
+|[Reaction]|0 AP*|
+
+__[Immediate Actions] and [Reactions]:__ These special kinds of actions can be taken when it is not your turn. Anything that is an [Immediate Action] or [Reaction] will list the precise conditions under which it can be taken. While you can take any number of [Immediate Actions], many GMs will limit this to a reasonable number (say, 3). However, you can only ever take a single [Reaction] per [Round]. 
+
+## Common Actions
+
+### Standard Attack
+Also known as "attack." This is a [Standard Action]. Select one of your weapons you are currently wielding and an opponent you can target with that weapon. Make an attack roll. If you are using a martial weapon, you have to equal or exceed their AC to hit. If you are using a magical weapon, you have to equal or exceed their SR to hit (and use a spell attack roll).
+
+_Attack roll = 1d20 + Strength + weapon bonuses + misc. bonuses_  
+_Spell Attack Roll = 1d20 + Intelligence + weapon bonuses + misc. bonuses_
+
+If the natural result of the d20 - the number that lands facing up - is equal to or greater than your weapons critical threshold, you scored a critical hit. If the attack roll still was not high enough to hit your opponent, you still hit but it is only a regular hit.
+
+If your opponent currently has any [Miss Chances], check those now. Roll d%. If you roll less than or equal to their [Miss Chance], you hit. Otherwise, you miss. If you have a critical hit but would miss, you still hit but it is only a regular hit.
+
+Then, you roll damage, based on the weapon. If a critical hit, multiply this by your weapon's critical multiplier. Then apply any [Weakness] or [Resistance] the target has. Then they subtract the damage from their health.
+
+All told, the process looks like this:
+1. Choose weapon and target
+1. Attack roll
+1. Miss Chance
+1. Roll base damage
+1. Apply critical multiplier, [Weakness], and [Resistance]
+1. Target takes damage
+
+### Standard Defense
+A favorite of pacifists and defensive fighters. This involves using your weapon to interrupt enemy attacks, getting your shield in the way, or otherwise focusing on defense. This is a [Standard Action]. If you have a melee weapon, you may make an attack roll. Otherwise, roll 1d20 + initiative + one-half your level. Your AC and SR receive a +4 bonus until your next turn, but cannot exceed the results of this roll. This can only be used once per [Round].
+
+### Move
+This is a [Standard Action], allowing you to move up to your movement speed. When moving on a square grid, the first diagonal step as part of this movement counts as 5 feet, and every diagonal step after counts as 10 feet.
+
+### 5-Foot Step
+This is a [Fast Action]. You move 5 feet in any direction. You may only take one 5-foot step per turn.
+
+### Cast a Spell
```

Comment:
Clarify multi-casting rules, as well as the requirement to advance all castings every turn (but option to advance multiple times in one turn).